website/settings/defaults.py

```diff
@@ -228,7 +228,13 @@ def parent_dir(path):
 ALL_MY_REGISTRATIONS_ID = '-amr'
 ALL_MY_PROJECTS_NAME = 'All my projects'
 ALL_MY_REGISTRATIONS_NAME = 'All my registrations'
-NOTEWORTHY_LINKS_NODE = None
+
+# Most Popular and New and Noteworthy Nodes
+POPULAR_LINKS_NODE = '6824t'  # TODO Override in local.py in production.
```

Comment:
These should not have hardcoded values like this. Instead, the defaults should be `None` and you should do one of two things:

- Raise an error if this isn't set, causing the app to fail early. This can be done here: https://github.com/CenterForOpenScience/osf.io/blob/a5250be2dadabeec7551e255a75081fda8632798/website/settings/__init__.py#L20
- Treat these as a feature flag and hide the relevant widgets if these are set to `None`.